code/game/objects/items/tools/spess_knife.dm

```diff
@@ -0,0 +1,144 @@
+/obj/item/spess_knife
+	name = "spess knife"
+	desc = "An all-in-one tool combining knife, screwdriver, cutters and many more."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "spess_knife"
+	inhand_icon_state = "screwdriver" // TODO
+	worn_icon_state = "screwdriver" // TODO
+	belt_icon_state = "screwdriver" // TODO
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	resistance_flags = FIRE_PROOF
+	tool_behaviour = null
+	toolspeed = 2 // Two times worse than default tools
```

Comment:
I understand that you don't want this to be OP and just a funny gimmick item, but is it *really* necessary to make it twice as slow as normal? It's maintenance loot which is in practice pretty damn rare to find but I'm expecting that people will just drop it when they realize just how slow it is at actually doing things that take time, such as window deconstruction, which is a sad fate for a cool item like this. What I'm saying is don't let 'balance' be the enemy of cool.

Reply:
It just makes sense that it's inconvenient to use compared to singular tools, and also it's a tiny item when folded, that can be hidden in boots. I think it's pretty good as it is, given that you don't care about toolspeed when you only need to hack airlocks.

Reply:
I think that SS13 should be designed with logical approach first, rather than gameplay balance. We have so much stuff that it's hard to learn what item does what. So the best way to design items IMO is to make people guess correctly what they do by glance.

IRL the swiss knife is a perfect example of jack of all trades, master of none. I don't care if people won't use it because it's not good enough for their meta game.

Reply:
After checking other tools, 2 is indeed too bad. Made it only 25% worse.